lib/galaxy/workflow/modules.py

```diff
@@ -1760,6 +1761,23 @@ def from_cwl(value):
         for key, value in execution_state.inputs.items():
             step_state[key] = to_cwl(value)
 
+        if when_expression is not None:
+            from cwltool.expression import do_eval
```

Comment:
Let's import this at the top

Reply:
You can tell I have developed habits stemming from frequent rebasing, imports at top result in conflicts 😆 .